src/shared/hooks/getPage.jsx

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+
+export default async function getPage(pageToGet) {
+    const peopleCollection = new Map(JSON.parse(localStorage.getItem("peopleData")))
+
```

Comment:
The data from this API didn't seem dynamic, so I decided to cache the data in localStorage once it gets it. It only caches the data asked for, to use as future reference instead of making a new API call, this also allows the app to be a little quicker at loading.